### DISCUSSION TITLE
关于通知插件建议

### DISCUSSION THREAD

**Discussion by lingdu990130556** [vuepress-theme-hope/vuepress-theme-hope#3173] [Q&A] [answered]
建议通知插件可以类似最小化和最大化按钮可以自由收起和打开：
![image](https://github.com/vuepress-theme-hope/vuepress-theme-hope/assets/76582458/708a6f2b-a286-4c38-900d-0a364284bd1c)

**Comment by Mister-Hope** [comment, marked as answer]
The notice component is called a **Notice** component, not a **Modal** component. From the screenshot, usage like this is output scope for the component.

If you want to pop notice with long content, fullscreen mode should be fine in most cases
